packages/react-components/package.json

```diff
@@ -33,7 +33,8 @@
     "@leapwallet/elements": "0.12.1",
     "chain-registry": "1.28.0",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "react-icons": "^5.2.1"
```

Comment:
Can we use something similar from https://heroicons.com/ and copy them into packages/react-components/src/lib/icons? Just trying to avoid adding dependencies and bundle size where possible. Chain-registry and Leap Elements already push the bundle size of this lib over 3MB which really pushes it, I'd like to figure out how to trim this down at some point.